extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -567,8 +568,8 @@ export class AmpSlideScroll extends BaseSlides {
     this.showSlide_(newIndex);
 
     const name = 'slideChange';
-    const detail = {index: newIndex};
-    const event = new CustomEvent(`slidescroll.${name}`, {detail});
+    const event =
+        customEvent(this.win, `slidescroll.${name}`, {index: newIndex});
```

Comment:
won't `name` always be slideChange here? If so, any idea why the name was being formatted into the type?